types/stripe-js/index.d.ts

```diff
@@ -655,6 +665,29 @@ declare module '@stripe/stripe-js' {
     | {token: Token; error?: undefined}
     | {token?: undefined; error: StripeError};
 
+  interface WrapperLibrary {
+    /**
+     * Your library’s name
+     */
```

Comment:
Can you flesh out the documentation comments for these fields? Doesn't need to be the full documentation, just focusing on what a developer might want to see in a tooltip.
- are there restrictions on this field, can it include whitespace/punctuation/etc?
- is there a doc page we can link to?

Reply:
I'll add more comments for them! The doc page PR is still under review and I'm thinking of merging this PR first to unblock any open source libraries who need to use this method.